component/libfuse/libfuse_wrapper.h

```diff
@@ -130,6 +131,11 @@ extern int libfuse_chown(char *path, uid_t uid, gid_t gid, fuse_file_info_t *fi)
 extern int libfuse_utimens(char *path, timespec_t tv[2], fuse_file_info_t *fi);
 #endif
 
+extern int blobfuse_cache_update(char* path);
+static int native_read_file(char *path, char *buf, size_t size, off_t, fuse_file_info_t *fi);
+static int native_write_file(char *path, char *buf, size_t size, off_t, fuse_file_info_t *fi);
+static int native_flush_file(char *path, fuse_file_info_t *fi);
+static int native_close_file(char *path, fuse_file_info_t *fi);
```

Comment:
does this need to be defined?

Reply:
C standards, you need to declare the function before you use. As functions bodies are at the end of the file while the registration is before that we need to declare the functions. Other way is pull up all those definitions before the registration logic. Wanted code to be clean so defined them towards the end of the file.

Reply:
I mean I see no native_close_file method defined, so do we need to define it here?